mailbox/tools/indexer/src/main/java/org/apache/mailbox/tools/indexer/ReIndexerPerformer.java

```diff
@@ -50,6 +50,7 @@
 
 import com.google.common.collect.ImmutableList;
 
+import fj.data.Either;
```

Comment:
As discussed during the daily we might consider adopting instead https://www.javadoc.io/doc/io.vavr/vavr/0.9.0/io/vavr/control/Either.html VAVR Either version.

Maybe that is the occasion for you to start posting to the James DEV message list to start discussing which library we should depend on to have "Either".